lib/services/project-data-service.ts

```diff
@@ -124,5 +124,13 @@ export class ProjectDataService implements IProjectDataService {
 			projectFilePath
 		};
 	}
+
+	@exported("projectDataService")
+	public getNsConfigDefaultContent(data: Object): string {
```

Comment:
can you please add the method to the lib tests: https://github.com/NativeScript/nativescript-cli/blob/master/test/nativescript-cli-lib.ts#L19